packages/kernel-6.1/6001-lustre-fix-Werror-enum-int-mismatch.patch

```diff
@@ -1,35 +0,0 @@
-From 82032af947c7a8b17ec30483a6d4a2d44759561f Mon Sep 17 00:00:00 2001
```

Comment:
Was this included in the new lustre patch that was added?

Reply:
Confirmed that it was included in the patch `0220-AL2023-6.1-Update-lustrefsx-to-2.15.6-fsx17-commit.patch`